app/assets/src/components/SampleUpload.jsx

```diff
@@ -599,8 +626,9 @@ class SampleUpload extends React.Component {
               <p className='upload-question'>
                 Want to upload multiple samples at once? <a href='/samples/bulk_new'>Click here.</a>
                 <br/>
-                Rather use our CLI? <a href='https://github.com/chanzuckerberg/idseq-web/blob/master/README.md#submit-a-sample' target='_blank'>Read documentation here.</a>
+                Rather use our CLI? <a className="modal-trigger" href='#cli_modal'>Instructions here.</a>
```

Comment:
maybe explain what CLI is to the general public?

Reply:
ok!